### PR TITLE
Refactor get metadata

### PR DESCRIPTION
**Pull request by tsmith023**
Implement specific options classes for getting objects by ID and generally that were covered by the inappropriate MetadataGet class.

### CODE REVIEW

File: weaviate/collection/classes.py [hunk 1]
```diff
@@ -547,6 +547,26 @@ def to_dict(self) -> Dict[str, Any]:
...
+
+
+class GetObjectByIdIncludes(IncludesModel):
+    classification: bool = False
+    vector: bool = False
+
+
+class GetObjectsIncludes(IncludesModel):
+    classification: bool = False
+    featureProjection: bool = Field(False, alias="feature_projection")
```
Comment (dirkkul) [inline weaviate/collection/classes.py:566]: I'm pretty sure that classification and feature projection are not yet included in GRPC - what happens if you set them to true? I assume that it is just ignored?

File: weaviate/collection/collection.py [hunk 2]
```diff
@@ -7,8 +7,9 @@
 from weaviate.collection.classes import (
     CollectionConfig,
     Errors,
+    GetObjectByIdIncludes,
+    GetObjectsIncludes,
     _MetadataReturn,
-    MetadataGet,
```
Comment (dirkkul) [inline weaviate/collection/collection.py:11]: you can probably delete the class

**Review by dirkkul — APPROVED** [review]

File: weaviate/collection/classes.py (hunk 1 above)
Comment (tsmith023) [reply weaviate/collection/classes.py:566]: These includes options are used when sending GET requests to `/v1/objects` so don't touch the gRPC. Though, this seems to be highlighting a potential naming convention conflict/confusion 🤔

Comment (dirkkul) [reply weaviate/collection/classes.py:566]: ah right

yeah that is a bit unfortunate that the have different default options on what they send back.

Comment (dirkkul) [reply weaviate/collection/classes.py:566]: let's discuss directly, this PR doesn't make it more confusing than before :)

**Pull request merged**